test/kickstart-templates/includes/post-containers.cfg

```diff
@@ -51,5 +51,5 @@ cat > /etc/containers/policy.json <<EOF
 EOF
 
 cat >> /etc/hosts <<EOF
-REPLACE_VM_BRIDGE_IP REPLACE_MIRROR_HOSTNAME
+REPLACE_VM_BRIDGE_IP REPLACE_MIRROR_HOSTNAME bootc-registry
```

Comment:
- Can we add a comment explaining what we're trying to do and why?
- Do we want to limit this setting to IPv6 case, or we'd better change [this template](https://github.com/openshift/microshift/blob/main/test/kickstart-templates/includes/main-ostreecontainer.cfg#L15)? 

What you're doing here might obsolete BOOTC_REGISTRY_URL setting completely.

Reply:
This was just a quick test (was using the same branch for my own local tests), it is changed to something (hopefully) simpler and cleaner now.